client/data/disputes/hooks.ts

```diff
@@ -30,6 +36,8 @@ export const useDisputeEvidence = () => {
 	return { updateDispute };
 };
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Return types is indeed needed: see https://github.com/Automattic/woocommerce-payments/pull/2350/files#diff-7c04a34030c29899ab88e563682fd1b13328caf74791625c5e656d30de91c26cR74 for some clues

Also at line 34,